src/configs/sys_config.py

```diff
@@ -0,0 +1,14 @@
+# System Configuration
+system_config = {
+    "num_users": 4,
+    "experiment_path": "./experiments/",
+    "dset": "cifar10",
+    "dump_dir": "./expt_dump/",
+    "dpath": "/data/unagi0/anakewat/imgs/",
+    "seed": 2,
+    "device_ids": {}
+    "dataset_splits": {
+        "iid": {"samples_per_user": 500},
```

Comment:
only one dataset split should be present in a single configuration. You can make two separate configuration if you want to represent both configurations.